### DISCUSSION TITLE
What will be happening the next couple of weeks

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#79] [Announcements]
[The major build system PR](https://github.com/tenacityteam/saucedacity/pull/66) has been merged. Since this has been done, this means several things:

* We can now start on a new Qt branch now that Conan has been removed.

* I will begin renaming Saucedacity to Tenacity and will start using Tenacity's icons.

We will also move to Codeberg, but since all three of our preferred organization names are already taken as placeholders, we cannot currently do this. We are in talks with Codeberg about this, and when we are able to move over, we will do so immediately. **For now, we will have to remain on GitHub**.

Looking forward to what's ahead! :D

**Comment by generic-pers0n** [comment]
There's also more to this announcement than originally planned, but just some clarifications for what some might now be seeing.

# Archiving Saucedacity
There is a new repository, https://github.com/tenacityteam/saucedacity-legacy, which contains all of Saucedacity's history and different version branches from 1.0 to 1.2.1. The repository will also include Saucedacity up to commit `405cd52`, the commit after the build system PR.

# What will happen to this repository
Once rebranding is finished, this repository will be renamed to 'tenacityteam/tenacity', and Tenacity development will restart again!

---

Those are all of the clarifications for right now. If there is anything else I must add, I will add it here.

**Comment by CodeCubeNeo** [comment]
Awesome! Really looking forward.
